PostHog/posthog
# feat: Duplicate a filter

## Problem
- addresses #11140 

- addresses rows that are not in a popup, still need to figure out where popup rows are in the product 😞 


## Changes
<img width="200" alt="Screen Shot 2022-08-11 at 1 12 17 PM" src="https://user-images.githubusercontent.com/17347501/184193532-e9d69b53-651f-4471-8de9-feb884fc90b7.png">
<img width="200" alt="Screen Shot 2022-08-11 at 1 15 20 PM" src="https://user-images.githubusercontent.com/17347501/184193987-2b18fb48-0db9-458f-b202-6e11bf20ce69.png">



## How did you test this code?

- create a filter in insights
- click the copy icon, a duplicate filter should be made
- delete the first icon and make another filter, should duplicate
- add more than one filter, than duplicate the second and first filters
